packages/oui-field/src/index.spec.js

```diff
@@ -466,6 +520,40 @@ describe("ouiField", () => {
             });
         });
 
+        describe("with validation", () => {
+            it("should retrieve custom error messages", () => {
+                const message = "Username must be a least 6 characters.";
```

Comment:
(optional but should not be) I don't remember if it has been done before but I think you should also test that interpolation works in custom messages.

Reply:
This could be done in a new test, only for testing the interpolation.